aten/src/ATen/cuda/CUDABlas.h

```diff
@@ -94,6 +94,48 @@ void bgemm<at::Half>(CUDABLAS_BGEMM_ARGTYPES(at::Half));
 template <>
 void bgemm<at::BFloat16>(CUDABLAS_BGEMM_ARGTYPES(at::BFloat16));
 #endif
+
+#define CUDABLAS_TRSM_ARGTYPES(Dtype)                                  \
+  cublasHandle_t handle, cublasSideMode_t side, cublasFillMode_t uplo, \
+      cublasOperation_t trans, cublasDiagType_t diag, int m, int n,    \
+      const Dtype *alpha, const Dtype *A, int lda, Dtype *B, int ldb
+
+template <typename Dtype>
+inline void trsm(CUDABLAS_TRSM_ARGTYPES(Dtype)) {
+  AT_ERROR("at::cuda::blas::trsm: not implemented for ", typeid(Dtype).name());
```

Comment:
AT_ERROR is deprecated. Use TORCH_CHECK for errors that can be triggered by a user (TORCH_CHECK(false, ...), for example) or TORCH_INTERNAL_ASSERT for internal logic failures (things that should never fail no matter the user input).